tests/ecosystem/upgrade/test_logging_upgrade.py

```diff
@@ -0,0 +1,234 @@
+import logging
+
+import pytest
+
+from ocs_ci.ocs.cluster import CephCluster, CephHealthMonitor
+from ocs_ci.ocs.resources.csv import CSV
+from ocs_ci.utility.deployment_openshift_logging import (
+    check_health_of_clusterlogging, get_clusterlogging_subscription
+)
+from ocs_ci.ocs.exceptions import CommandFailed
+from ocs_ci.framework import config
+from ocs_ci.ocs.resources.packagemanifest import PackageManifest
+from ocs_ci.framework.pytest_customization.marks import (
+    post_ocp_upgrade
+)
+from ocs_ci.ocs.resources.pod import get_all_pods
+from ocs_ci.ocs import constants, ocp
+from ocs_ci.utility.retry import retry
+from ocs_ci.utility.utils import (
+    get_ocp_version, run_cmd
+)
+
+logger = logging.getLogger(__name__)
+
+
+def check_cluster_logging():
+    """
+    Few checks to assert for before and after upgrade
+    1. Checks for pods in openshift-logging project
+    2. Checks for the health of logging cluster before upgrade
+
+    """
+
+    assert get_all_pods(
+        namespace=constants.OPENSHIFT_LOGGING_NAMESPACE
+    ), 'Some or all pods missing in namespace'
+
+    assert check_health_of_clusterlogging(), "Cluster is not Healthy"
+
+
+@retry(CommandFailed, 5, 30, 2)
```

Comment:
same comment with retry bellow. I prefer to avoid backoff to 2 and rather use 1 here.